src/math/Euler.js

```diff
@@ -9,6 +9,8 @@ import { _Math } from './Math.js';
  * @author bhouston / http://clara.io
  */
 
+var _matrix, _quaterion;
```

Comment:
This should be __quaternion_, right? :-)

Reply:
You are so right!^^